app/controllers/miq_capacity_controller.rb

```diff
@@ -95,7 +95,7 @@ def planning
                  (vm_opts[:vcpus] && @sb[:planning][:options][:trend_vcpus]) ||
                  (vm_opts[:memory] && @sb[:planning][:options][:trend_memory]) ||
                  (vm_opts[:storage] && @sb[:planning][:options][:trend_storage])
-            add_flash(_("At least one %s must be selected") % "VM Options", :error)
+            add_flash(_("At least one VM Options must be selected"), :error)
```

Comment:
`VM Options` should probably be singular `VM Option`.  Seems it was wrong before this PR.